cotainr/util.py

```diff
@@ -13,6 +13,8 @@
     Run a the command described by `args` while streaming stdout and stderr.
 get_systems()
     Get a dictionary of predefined systems, defined in systems.json
+answer_yes
```

Comment:
```suggestion
answer_is_yes()
```

... but it's debatable whether there's any value in documenting this function separately here in the module docstring 🤷

Reply:
Done.